src/gameComponents/Areas.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div id="good" ref="good" :class="lastStyle['good']">
+    <div id="excellent" ref="excellent" :class="lastStyle['excellent']">
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+
+import Vue from 'vue'
+import Component from 'vue-class-component'
+import { IAreasPositions } from '@/types'
+import { Emit, Watch } from 'vue-property-decorator'
+import { IStateScore } from '@/store/modules/score/typesScore'
+
+@Component
+export default class Areas extends Vue {
+  animationStyle: {[index: string]: string} = {
+    good: '',
+    excellent: ''
+  }
+
+  mapStyles: {[index: string]: string} = {
+    good: 'good-animation',
+    excellent: 'excellent-animation'
+  }
+
+  get lastStyle () {
```

Comment:
The same question. Do you need this getter, if it just returns data?

Reply:
@silchencko I'll check it too